app/javascript/mastodon/locales/en.json

```diff
@@ -2,6 +2,7 @@
   "account.block": "Block @{name}",
   "account.block_domain": "Hide everything from {domain}",
   "account.blocked": "Blocked",
+  "account.direct": "Direct Message @{name}",
```

Comment:
I would prefer "Direct message @{name}" (small m), precedents: "Edit profile" (small p)

Reply:
oh! sure

Also, how does the localisation system work exactly? I don't know all the languages supported, so can't translate for them

Reply:
You just add the English string. Volunteers add translations using weblate.joinmastodon.org or other PRs. `yarn manage:translations` just minimizes code diff later because it makes sure all files are formatted and ordered the same.

Reply:
okay, cool; I'll rebase and re-run `yarn manage:translations`

Reply:
FWIW i wouldn't be worried about the casing on files other than en.json. It'll get translated eventually anyway. So you can save yourself that trouble imo.

Reply:
That's kinda why I did it as a separate commit.